lithium/test_lithium.py

```diff
@@ -510,6 +511,29 @@ def test_char_dd(self):
         self.assertEqual(t.parts, [b"d", b"a", b"t", b"a", b"\n", b"2"])
         self.assertEqual(t.after, b"\nDDEND\npost\n")
 
+    def test_jsstr(self):
```

Comment:
I eventually realised in #48 that the functions in the test files seem to be fairly straightforward when being named and that they should not be used by other files/repos (since they are test-related).

Thus, going forward, I probably wouldn't dwell on this suppression for `test_lithium.py`.